src/components/input/input.js

```diff
@@ -679,16 +694,17 @@ function mdMaxlengthDirective($animate, $mdUtil) {
       });
     });
 
-    function renderCharCount(value) {
-      // If we have not been initialized or appended to the body yet; do not render
+    function renderCharCount() {
+      // If we have not been initialized or appended to the body yet; do not render.
       if (!charCountEl || !charCountEl.parent) {
```

Comment:
Not directly related to this PR, but this check seems incorrect. `charCountEl.parent` will always be truthy if `charCountEl` is truthy. Maybe the intention was `!charCountEl.parent()`?